thoth/storages/graph/postgres.py

```diff
@@ -1094,6 +1094,63 @@ def get_error_solved_python_package_versions_all(
 
             return query.all()
 
+
+    def get_error_solved_document_id_all(
+        self,
+        package_name: str = None,
+        package_version: str = None,
+        index_url: str = None,
+        *,
+        unsolvable: bool = False,
+        unparseable: bool = False,
+        start_offset: int = 0,
+        count: Optional[int] = DEFAULT_COUNT,
+        os_name: str = None,
+        os_version: str = None,
+        python_version: str = None,
+        distinct: bool = False,
+        limit_results: bool = True
+    ) -> List[str]:
+        """Retrieve solver document id with error Python package versions in Thoth Database.
+
+        if unsolvable=True -> get_unsolvable_python_package_versions
+        if unparseable=True -> get_unparseable_python_package_versions
+
+        Examples:
+        >>> from thoth.storages import GraphDatabase
+        >>> graph = GraphDatabase()
+        >>> graph.get_error_solved_document_id_all()
+        ['solver-fedora-32-py37-324232']
+        """
+        os_version = OpenShift.normalize_os_version(os_name, os_version)
```

Comment:
Normalization of `package_name` missing.

Reply:
normalization is done inside `_construct_error_solved_python_package_versions_query`

Reply:
Ack :+1: